design-library/source/_patterns/00-global/03-fonts.md

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
It would be good to put this into a "03-fonts" pattern directory under "00-global"